LayoutTests/accessibility/text-marker/text-marker-bidi-element-arabic-expected.txt

```diff
@@ -0,0 +1,196 @@
+This test verifies that a combination of Arabic and English is read correctly by Accessibility, when using CSS unicode-bidi property.
+
+PASS: rangeLength === 28
+PASS: maxBounds === 188
```

Comment:
Maybe delete this line. It looks like it's failing on the bots because we don't get exactly the same number of pixels with and without ITM.